pkg/inventory/type-conv.go

```diff
@@ -4,11 +4,22 @@
 package inventory
 
 import (
+	"fmt"
+
 	"k8s.io/apimachinery/pkg/runtime/schema"
 	"sigs.k8s.io/cli-utils/pkg/apis/actuation"
 	"sigs.k8s.io/cli-utils/pkg/object"
+	"sigs.k8s.io/controller-runtime/pkg/client"
 )
 
+// ObjMetadataEqualObjectReference compares an ObjMetadata with a ObjectReference
+func ObjMetadataEqualObjectReference(id object.ObjMetadata, ref actuation.ObjectReference) bool {
```

Comment:
This helper function looks pretty independent. It could be in a separate PR.

In fact, this whole file seems like helper funcs that could be factored into a separate PR.